encoder/libav_encoder.cpp

```diff
@@ -387,8 +447,12 @@ void LibAvEncoder::initOutput()
 	char err[64];
 	if (!(out_fmt_ctx_->flags & AVFMT_NOFILE))
 	{
-		std::string filename = options_->output;
-
+		std::string filename = options_->output;			
+		char subfilename[256];
+		snprintf(subfilename, sizeof(subfilename), options_->output.c_str(), segment_num);
+		filename = subfilename;
+		printf("Outputting with filename: %s\n", filename.c_str());
```

Comment:
Just wanted to check whether printf is the right thing here, or whether it should be some kind of logging statement?